layers/+source-control/version-control/packages.el

```diff
@@ -54,7 +54,8 @@
       ;; If you enable global minor mode
       (when (and (eq version-control-diff-tool 'git-gutter)
                  version-control-global-margin)
-        (global-git-gutter-mode t))
+        (run-with-idle-timer 5 nil (lambda ()
```

Comment:
I suppose I could use `:if EXPR`, but that would change the behavior slightly (people could, if they wanted to, use `git-gutter` in some buffers and `git-gutter+` in others. If we were going to make it conditional, that could also be done in `version-control-packages`. Thoughts?